CHANGELOG.md

```diff
@@ -36,11 +37,12 @@
    `NodeId` of both the source and destination nodes of a channel (#1133).
 
 ## Bug Fixes
- * Delay disconnecting peers if we receive messages from them even if it takes
-   a while to receive a pong from them. Further, avoid sending too many gossip
-   messages between pings to ensure we should always receive pongs in a timely
-   manner. Together, these should significantly reduce instances of us failing
-   to remain connected to a peer during initial gossip sync (#1137).
+ * Previous versions would often disconnect peers during initial graph sync due
+   to ping timeouts while processing large numbers of gossip messages. We now
+   delay disconnecting peers if we receive messages from them even if it takes
+   a while to receive a pong from them. Further, we avoid sending too many
+   gossip messages between pings to ensure we should always receive pongs in a
+   timely manner (#1137).
  * If a payment is sent, creating an outbound HTLC and sending it to our
    counterparty (implying the `ChannelMonitor` was persisted on disk), but the
    `ChannelManager` was not persisted prior to shutdown/crash, no
```

Comment:
My feedback was more about the second bullet just describing the bug and not the fix. Maybe that's ok but the use of the word "will" instead of "would" makes it seem like it's describing the fix.